### PR TITLE
feat(rate-limit): add redis to hub and sequencer

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -196,6 +196,7 @@ services:
     env_file:
       - ${PINEAPPLE_REPO_PATH}/.env
       - ./envs/.env.pineapple
+      - ./envs/.env.pineapple.local
```
Comment (wa0x6e) [inline docker-compose.yml:199]: What’s the difference with the one above?

Comment (Todmy) [reply docker-compose.yml:199]: it's the file that is ignored by git and contains sensitive information. It's done to prevent keys leakage.

Comment (wa0x6e) [reply docker-compose.yml:199]: Why isn't the .local version systematically added everywhere ?

Comment (Todmy) [reply docker-compose.yml:199]: because we don't need it everywhere. But holding and sharing empty files doesn't make sense for me

**Pull request merged**